package.json

```diff
@@ -23,6 +23,7 @@
     "lib",
     "cli.js"
   ],
+  "typings": "./d.ts/index.d.ts",
```

Comment:
@jbucaran Sure, see [here](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html). It is just an alias for `types` that allows TypeScript's compiler to get suitable declaration file for package. If you won't do it, you need manually load your declaration from package with triple-slash directive.